client/state/happychat/selectors.js

```diff
@@ -42,15 +42,15 @@ export const getHappychatTranscriptTimestamp = state => (
  * @return {String} current state value
  */
 export const getHappychatConnectionStatus = createSelector(
-	state => state.happychat.connectionStatus
+	state => get( state, 'happychat.connectionStatus' )
```

Comment:
From [the reducer side](https://github.com/Automattic/wp-calypso/blob/master/client/state/happychat/reducer.js#L153), it looks like defaulting to `'uninitialized'`. As there are more and more places reusing these state strings, ( e.g. isHappychatClientConnected` ), it might be a good timing to add those to `constants.js`, so we have a unified place to access them.